### ISSUE TITLE
Entry/Code expiration

### ISSUE THREAD

**Issue by cdelahousse** [SteamedPears/Code-Review#66]
I was checking out Pastebin for ideas. On their front page, they have a expiration setting that allows a user to set a time in the future when the code snippet gets taken offline.

Sounds like a good idea.

**Comment by bheesham** [comment]
Yeah, I was thinking about that. It's pretty nice, because we can have a default of a month or so.
I don't think it should expire X days after creation. It should expire X days after the last activity. That gives people more time to do stuff.
